.github/workflows/test.yml

```
@@ -0,0 +1,33 @@
+name: "test"
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        luaVersion: ["5.1", "5.2", "5.3", "5.4", "LuaJIT-2.1.0-beta3", "luajit-openresty"]
+
+    steps:
+    - uses: actions/checkout@master
+    - uses: leafo/gh-actions-lua@master
```

```suggestion
    - uses: actions/checkout@v2

    - uses: leafo/gh-actions-lua@v8
```

Calling out these tagged versions rather than using `master` is important for two reasons. First it enables use of cached tooling rather than having to build the toolchain on use each time. Second it cuts down on breakage when they change. Following the major version only not pitting the minor & patch version is a decent compromise to get these two benefits.